R/FilterStateRange.R

```diff
@@ -1,16 +1,24 @@
+# RangeFilterState ------
+
 #' @name RangeFilterState
-#' @title `FilterState` object for numeric variable
-#' @description Manages choosing a numeric range
 #' @docType class
-#' @keywords internal
 #'
+#' @title `FilterState` object for numeric variable
+#'
+#' @description Manages choosing a numeric range.
 #'
 #' @examples
-#' filter_state <- teal.slice:::RangeFilterState$new(
+#' # use non-exported function from teal.slice
+#' include_js_files <- getFromNamespace("include_js_files", "teal.slice")
+#' include_css_files <- getFromNamespace("include_css_files", "teal.slice")
+#' include_js_files <- getFromNamespace("include_js_files", "teal.slice")
```

Comment:
why include_js_files <- getFromNamespace("include_js_files", "teal.slice") is repeated twice? cc @kartikeyakirar